README.rst

```diff
@@ -81,21 +81,23 @@ Installation on Open edX Devstack
 Compatibility Notes
 --------------------
 
-+-------------------+-----------------+
-| Open edX Release  |  Version        |
-+===================+=================+
-|       Ironwood    |   <  3.0        |
-+-------------------+-----------------+
-|       Juniper     |   >= 3.0 < 5.0  |
-+-------------------+-----------------+
-|        Koa        |   >= 4.9 < 6.0  |
-+-------------------+-----------------+
-|       Lilac       |   >= 4.9 < 6.0  |
-+-------------------+-----------------+
-|       Maple       |   >= 6.0        |
-+-------------------+-----------------+
-|       Nutmeg      |   >= 7.0        |
-+-------------------+-----------------+
++------------------+--------------+
+| Open edX Release | Version      |
++==================+==============+
+| Ironwood         | < 3.0        |
++------------------+--------------+
+| Juniper          | >= 3.0 < 5.0 |
++------------------+--------------+
+| Koa              | >= 4.9 < 6.0 |
++------------------+--------------+
+| Lilac            | >= 4.9 < 6.0 |
++------------------+--------------+
+| Maple            | >= 6.0       |
++------------------+--------------+
+| Nutmeg           | >= 7.0       |
++------------------+--------------+
+| Olive            | >= 8.0       |
```

Comment:
Maybe it is a good idea to eliminate the backends that are not still useful for this release (maple, nutmeg, or olive), for example:
[coursekey_h_v1.py](https://github.com/eduNEXT/eox-core/blob/MJG/DS-364-olive-compatibility/eox_core/edxapp_wrapper/backends/coursekey_h_v1.py)
[enrollment_h_v1.py](https://github.com/eduNEXT/eox-core/blob/MJG/DS-364-olive-compatibility/eox_core/edxapp_wrapper/backends/enrollment_h_v1.py)
[pre_enrollment_h_v1.py](https://github.com/eduNEXT/eox-core/blob/MJG/DS-364-olive-compatibility/eox_core/edxapp_wrapper/backends/pre_enrollment_h_v1.py)
[third_party_auth_j_v1.py](https://github.com/eduNEXT/eox-core/blob/MJG/DS-364-olive-compatibility/eox_core/edxapp_wrapper/backends/third_party_auth_j_v1.py)
[users_h_v1.py](https://github.com/eduNEXT/eox-core/blob/MJG/DS-364-olive-compatibility/eox_core/edxapp_wrapper/backends/users_h_v1.py)
[users_j_v1.py](https://github.com/eduNEXT/eox-core/blob/MJG/DS-364-olive-compatibility/eox_core/edxapp_wrapper/backends/users_j_v1.py)

If you find a backend that is not used by a release after lilac, I think it's worth removing it.

Reply:
Each file means: support from release X until new release file Y is created. IMO removing them is not the solution but renaming them using the oldest release we want to support.

Reply:
what do you folks think? @eduNEXT/dedalo

Reply:
what about the ones without maple, nutmeg or olive versions? eg. bearer_authentication_j_v1.py

Reply:
I proposed to remove it because the olive release is going to support only maple, nutmeg, and olive, and for example: the users' backend have juniper, lilac, and maple, but with the users_m we can support maple, nutmeg and olive. With the third_party_auth backend you have a juniper and lilac version, but with lilac we can support maple, nutmeg and olive, we don't need third_party_auth in juniper for this release. And the same with the examples I sent. It is a proposal to clean the backends we are not going to use in this release.

Reply:
I didn't say anything about bearer_authentication_j_v1.py because we still need it for this release. But if you want to change the naming, you must update the readme and the [settings files](https://github.com/eduNEXT/eox-core/blob/MJG/DS-364-olive-compatibility/eox_core/settings/common.py#L22). I don't think changing the naming, in this case, its necessary but I let it in your hands.

Reply:
You're right; it's a good idea to remove old backends. But maybe we should do it all together, including the ones without newer versions (either way, we know those old backends work with newer versions. That's why they haven't been updated)

Reply:
@mariajgrimaldi to upgrade a major version, you have to make a commit with ``perf:`` key

Reply:
@MaferMazu: I added the backend changes but the PR got too big for the labeler. I'll open another PR

Reply:
@MaferMazu: [here](https://github.com/eduNEXT/eox-core/pull/245) it is

Reply:
Thanks @mariajgrimaldi I'm going to resolve this comment in favor of you taking into account every feedback here.